File: js/ordenacao-simples.js
let ordenacao_simples = () => {
    let lista_devalores = [];
    for (let i = 0; i < 10; i++) {
        let valor_entrada = parseInt(
            prompt("Por favor digite o valor a ser ordenado ")
        );
        lista_devalores.push(valor_entrada);
    }
    lista_devalores.sort((a, b) => {
        if (a > b) return 1;
        if (a < b) return -1;
        return 0;
    });
    console.log(`
        a lista de valores ordenadas é ${lista_devalores}\n
        a lista de valores reversa é ${lista_devalores.reverse()}
    `);
};

ordenacao_simples();

File: python/ordenacao-simples.py
def ordenacao_simples():
    lista_devalores = []
    for i in range(10):
        valor_entrada = int(input("Por favor digite o valor a ser ordenado "))
        lista_devalores.append(valor_entrada)
    lista_devalores_ordenados = list(sorted(lista_devalores))
    lista_devalores_invertidos = list(reversed(lista_devalores_ordenados))
    print(
        f'a lista de valores ordenadas é {lista_devalores_ordenados}' +
        f'\na lista de valores invertida é {lista_devalores_invertidos}'
    )


ordenacao_simples()
